services/cartService.js

```diff
@@ -0,0 +1,20 @@
+const cartDao = require('../models/cartDao');
+
+const addToCart = async (productId, userId) => {
+  const isFirst = await cartDao.checkIfFirst(productId, userId);
+
+  if (isFirst) {
+    return cartDao.addToCart(productId, userId);
+  } else {
+    return cartDao.plusItemCount(productId, userId);
+  }
+};
```

Comment:
```suggestion
const addToCart = async (productId, userId) => {
  return cartDao.createOrUpdateCart(productId, userId, quantity)
};
```
구두로 설명드린 부분이고, sql문은 dao쪽 코멘트에서 남겨드리겠습니다.